### PR TITLE
Update hooks.md - fixed URL to repo

### PR DESCRIPTION
**Pull request by karpikpl**
URL changed from "lib' to "src"

### CODE REVIEW

**Comment by papb** [comment]
Hello! I see you are a first-time contributor, thank you for taking the time to help Sequelize! I hope to see more PRs from you in the future!

Please change the target of this PR from `master` to the `v6` branch :)

File: docs/manual/other-topics/hooks.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Hooks (also known as lifecycle events), are functions which are called before an
 
 ## Available hooks
 
-Sequelize provides a lot of hooks. The full list can be found in directly in the [source code - lib/hooks.js](https://github.com/sequelize/sequelize/blob/master/lib/hooks.js#L7).
+Sequelize provides a lot of hooks. The full list can be found in directly in the [source code - src/hooks.js](https://github.com/sequelize/sequelize/blob/master/src/hooks.js#L7).
```
Comment (papb) [inline docs/manual/other-topics/hooks.md:9]: I think the following will be more future-proof:

```suggestion
Sequelize provides a lot of hooks. The full list can be found in directly in the [source code - src/hooks.js](https://github.com/sequelize/sequelize/blob/v6/lib/hooks.js#L7).
```

**Review by papb — CHANGES_REQUESTED** [review]

**Comment by karpikpl** [comment]
need to start a new PR from v6...

**Comment by karpikpl** [comment]
sorry - I've created a new PR branched of v6

**Comment by papb** [comment]
@karpikpl No need to be sorry :) Thank you very much!!

**Pull request closed without merging**